contracts/hub/Hub.sol

```diff
@@ -272,6 +272,11 @@ contract Hub is IHub, HubUtils, OwnableUpgradeable, HubUpgradeable {
         emit ArchetypeSet(input[0]);
     }
 
+    function setMetadataUri(string memory metadataUri_) external {
```

Comment:
This should be `_metadataUri` to match conventions of the same argument within `initialize()`.